fud/fud/stages/interpreter.py

```diff
@@ -88,16 +88,17 @@ def _is_data_converter(self):
 
     def _define_steps(self, input_data, builder, config):
         script = config["stages", self.name, "exec"]
-        data_path = config["stages", "verilog", "data"]
+        data_path_exists: bool = (config["stages", "verilog", "data"] or
+                                  config.get(["stages", "mrxl", "data"]))
```

Comment:
It looks a little weird that the interpreter stage needs to know about the MrXL stage, but I see why: `config["stages", "verilog", "data"]` is unset at setup (`_define_steps`) time and will be populated later. One possible alternative could be to move the construction of `cmd` to run time (`def interpret(...):` below), but that unfortunately means that we can't see the full command in a dry-run… so maybe not!

Reply:
@susan-garry, is there a way to not change `interpreter.py`? I agree with @sampsyo that it's suboptimal to change this stage to support the mrxl.data abstraction